initial-promises/node-server/common/1.0/update.cf

```diff
@@ -318,11 +318,14 @@ body action u_background
 ############################################
 body classes success(if, else, kept)
 {
-        promise_kept => { "${kept}" };
-        promise_repaired => {"${if}"};
-        repair_failed => { "${else}" };
-        repair_denied => { "${else}" };
-        repair_timeout => { "${else}" };
+        promise_kept     => { "${kept}" };
+        promise_repaired => { "${if}"   };
+        repair_failed    => { "${else}" };
+        repair_denied    => { "${else}" };
+        repair_timeout   => { "${else}" };
+
+        cancel_kept      => { "${else}" };
+        cancel_repaired  => { "${else}" };
```

Comment:
I'm not sure it would be relevant. If we cancel the repaired class as well, we'll most likely hide the update that occured, and user may scratch their head on why promises are updated, but report kept